src/status_im2/contexts/emoji_picker/view.cljs

```diff
@@ -16,8 +16,7 @@
     [status-im2.contexts.emoji-picker.utils :as emoji-picker.utils]
```

Comment:
Not related to this pr but somewhat important to note. The concept of "Contexts" was for large feature chunks - e.g messaging, communities, onboarding etc.
Emoji picker should live in somewhere like status-im2.common.emoji-picker imo. We can make a follow up pr to move it. 👍

Reply:
yeah context and common getting messy, re-evaluation is needed, i remember @ilmotta had a vision how it should work

Reply:
Correct guys, `contexts` is just an abbreviation for *bounded contexts* from DDD. We're not doing DDD, but still, the idea is to separate domains because they tend to evolve in very different ways. We have a glossary and a link to explain what *context* means. `emoji-picker` as a bounded context makes no sense.

https://github.com/status-im/status-mobile/blob/4934a02f881b02c1ed3d76c028b052e3e3715609/doc/new-guidelines.md#glossary

Reply:
Yep, totally agree, let's move it in a follow up! :+1: